parallel-install/pkg/preinstaller/preinstaller.go

```diff
@@ -0,0 +1,127 @@
+package preinstaller
```

Comment:
It's good to describe the package here unless you want a separate go.doc file. Take a look at other packages for examples.

Reply:
Will do!